src/styles/main.scss

```diff
@@ -20,23 +20,14 @@ body {
   grid-row-gap: 51px;
 }
 
-@media screen and (max-width: 900px) and (min-width: 300px) {
-
-  .header {
-    height: auto;
-    flex-direction: column;
-  }
-  .nav {
-    flex-direction: column;
-  }
-
-  .navigation__item {
-    display: block;
-    width: 100%;
+@media screen and (min-width: 1201px) {
+  .main {
+    grid-template-columns: repeat(auto-fit, $card-width);
   }
+}
 
-  .logo {
-    display: block;
-    width: 100%;
+@media screen and (max-width: 800px) {
+  .main {
+    grid-template-columns: repeat(auto-fit, $card-width);
```

Comment:
It still doesn't work on big screens, maybe try without auto-fit. As I understand it, they want us to have four columns in large sizes.